docs/book/src/developer/providers/v1alpha4-to-v1beta1.md

```diff
@@ -22,6 +22,10 @@ The core ClusterAPI providers will support upgrade from v1alpha3 **and** v1alpha
 from v1alpha3 and v1alpha4 to v1beta1 have been implemented. If other providers also want to support the upgrade from v1alpha3 **and**
 v1alpha4, the same conversions have to be implemented.
 
+## Certificates
+
+The `serving-cert` certificates now have organization set to `k8s-sig-cluster-lifecycle`.
+
```

Comment:
Is it okay to have a section of its own?
None of the other sections seemed like a good place for this point.

Reply:
I think it's fine. I wonder if we should provide some context, e.g. link the corresponding issue